students/chelsea_nayan/lesson05/products.csv

```diff
@@ -0,0 +1,7 @@
+_id, description, product_type, quantity_available
```

Comment:
Not really a coding error, but leaving spaces in a CSV file will cause the CSV reader to pick up the space as part of the key, for example " description", which will cause this error:

```
  File "/Users/ldconejo/Documents/UW/SP_Python220B_2019/SP_Python220B_2019/students/chelsea_nayan/lesson05/database.py", line 69, in import_data
    'description': row['description'],
KeyError: 'description'
```